Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why is main.storyboard checked in ? Could you remove this ? Also can you use pinit to generated project and add your files there ?